Remove debugfs related sepolicy rules

ceiba1985 opened the pull request:
Google doesn't allow vendors to use debugfs in the user build
from android 12, and they add checks in some CTS test cases.
So the sepolicy rules related debugfs in celadon should be
removed.

Tracked-On: OAM-104124
Signed-off-by: jizhenlo <zhenlong.z.ji@intel.com>

sysopenci commented:
Expected syntax:
    Commit message one-line summary

      Extra paragraphs can be added here.

      Longer description that explains the problem / solution
      or whatever that needs to be said

      It can contain
        multiple lines
        multiple paragraphs
        blank lines
        references

      But each paragraph must be followed by a blank line

      However, you need to keep the 'Tracked-On', 'Change-Id',
      'Signed-off-by', ..., fields in the **lastest** paragraph of
      the commit message.

      Change-Id: I9e58f79c7faf7f0edb9792e8b5322ab161331141
      Tracked-On: https://jira.server.ext/browse/PROJECTA-123
      Signed-off-by: Developer Name <developer.name@intel.com>

sysopenci commented:
Expected syntax:
    Commit message one-line summary

      Extra paragraphs can be added here.

      Longer description that explains the problem / solution
      or whatever that needs to be said

      It can contain
        multiple lines
        multiple paragraphs
        blank lines
        references

      But each paragraph must be followed by a blank line

      However, you need to keep the 'Tracked-On', 'Change-Id',
      'Signed-off-by', ..., fields in the **lastest** paragraph of
      the commit message.

      Change-Id: I9e58f79c7faf7f0edb9792e8b5322ab161331141
      Tracked-On: https://jira.server.ext/browse/PROJECTA-123
      Signed-off-by: Developer Name <developer.name@intel.com>

ceiba1985 committed:
Remove debugfs related sepolicy rules

Google doesn't allow vendors to use debugfs in the user build
from android 12, and they add checks in some CTS test cases.
So the sepolicy rules related debugfs in celadon should be
removed.

Tracked-On: OAM-104124
Signed-off-by: jizhenlo <zhenlong.z.ji@intel.com>

sysopenci commented:
Expected syntax:
    Commit message one-line summary

      Extra paragraphs can be added here.

      Longer description that explains the problem / solution
      or whatever that needs to be said

      It can contain
        multiple lines
        multiple paragraphs
        blank lines
        references

      But each paragraph must be followed by a blank line

      However, you need to keep the 'Tracked-On', 'Change-Id',
      'Signed-off-by', ..., fields in the **lastest** paragraph of
      the commit message.

      Change-Id: I9e58f79c7faf7f0edb9792e8b5322ab161331141
      Tracked-On: https://jira.server.ext/browse/PROJECTA-123
      Signed-off-by: Developer Name <developer.name@intel.com>

sysopenci reviewed (commented):
PR integrated into 'preintegration' builder



More details, logs, and binaries: /absp/builders/celadon-preintegration/builds/4528

Done by "Ji, Zhenlong Z" <zhenlong.z.ji@intel.com>

sysopenci reviewed (commented):
Failure when executing preintegration builder

FAILURE: Trigger blocking tests Triggered: CIT-CIV-TGL-R(tgl_nuc:caas - userdebug)

More details, logs, and binaries: /absp/builders/celadon-premerge/builds/2588

sysopenci reviewed (commented):
PR integrated into 'premerge' builder



More details, logs, and binaries: /absp/builders/celadon-premerge/builds/2590

Done by "T Prabhu, Vignesh" <vignesh.t.prabhu@intel.com>

sysopenci reviewed (commented):
cactus finished building and testing your changes 

It was requested for merge by "T Prabhu, Vignesh" <vignesh.t.prabhu@intel.com> 

so we merge it. Thanks for your contribution.

more details, logs, and binaries: /absp/builders/celadon-mergerequest/builds/3011
